webapp/src/main/java/org/apache/atlas/web/rest/BusinessPolicyREST.java

```diff
@@ -112,4 +113,45 @@ public void unlinkBusinessPolicy(@PathParam("policyId") final String policyGuid,
             RequestContext.get().endMetricRecord(metric);
         }
     }
+
+
+    @POST
+    @Path("/move/{assetId}")
+    @Timed
+    public void moveBusinessPolicy(
+            @PathParam("assetId") String assetId,
+            final MoveBusinessPolicyRequest request
+    ) throws AtlasBaseException {
+        // Start performance metric recording
+        AtlasPerfMetrics.MetricRecorder metric = RequestContext.get().startMetricRecord("moveBusinessPolicy");
```

Comment:
We can use only perf logs here which is already enabled in lines below, and remove the metrics logs.

Reply:
so to trace individual method metrics we don't need metrics? Can perf log give a metric for all the methods used called from this as well?